config.example/host_vars/example01

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Would it be better if we made this gpu01? That would make it more consistent with the examples we have in the inventory file and as long as everything is commented out it shouldn't conflict with anything. I'd be worried that users will see "example01" and not necessarily correlate that to the host.

Reply:
Sure, this makes sense. I'll make this change throughout.